package.json

```diff
@@ -54,6 +56,9 @@
     "webpack": "4.43.0",
     "webpack-cli": "3.3.11"
   },
+  "engines": {
+    "node": ">=10.0.0"
+  },
```

Comment:
@keefertaylor , Does this seem right for the supported versions of Node? (`eslint-plugin-node` uses the `engines` field to determine if you're writing any code using features not supported in a version of Node you claim to support).